### PR TITLE
pipx: add page

### PR DESCRIPTION
**Pull request by anaynayak**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/pipx.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# pipx
+
+> Install and run python applications in isolated environments.
+> More information: <https://github.com/pipxproject/pipx>.
+
+- Run an app in a temporary virtual environment:
+
+`pipx run pycowsay moo`
```
Comment (navarroaxel) [inline pages/common/pipx.md:8]:
```suggestion
`pipx run {{pycowsay}} {{moo}}`
```

File: pages/common/pipx.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`pipx install {{package}}`
+
+- List installed packages:
+
+`pipx list`
+
+- Run an app in a temporary virtual environment with a pakage name different from the executable:
+
+`pipx run --spec httpx-cli httpx http://www.github.com`
```
Comment (navarroaxel) [inline pages/common/pipx.md:20]:
```suggestion
`pipx run --spec {{httpx-cli}} {{httpx}} {{http://www.github.com}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @anaynayak, thanks for your submission, I left some comments.

File: pages/common/pipx.md (hunk 1 above)
Comment (anaynayak) [reply pages/common/pipx.md]: Thanks. Fixed it.

File: pages/common/pipx.md (hunk 2 above)
Comment (anaynayak) [reply pages/common/pipx.md]: Thanks. Fixed it.

**Review by owenvoke — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**